<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDlg</class>
 <widget class="QFrame" name="NewDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>282</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>新建表单项</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLineEdit" name="editVariety">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>62</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>101</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>客户：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPrice">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>101</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>62</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>品种：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCustomer">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>101</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>101</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>单价：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>62</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>明细码：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSure">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>62</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboNo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>23</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelChoiceNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>23</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择编号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCode5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCode4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCode3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCode2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCode1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="editComment">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
